src/Normalizer/FieldItemListNormalizer.php

```diff
@@ -91,8 +94,27 @@ public function normalize($object, $format = NULL, array $context = []) {
           $markup = '<!doctype html><html><head><meta charset="utf-8" /></head><body>' . $markup . '</body></html>';
           @$document->loadHTML(Html::decodeEntities($markup));
 
-          // Note that we are putting the content of these fields into the body.
-          $this->transformer->transform($article, $document);
+          // Determine correct context for transformation.
+          $context = $article;
```

Comment:
Let's call it something other than `$context`, since that is already used as the third argument to this method. Maybe `$article_context`?